src/bin/lpython.cpp

```diff
@@ -902,6 +903,7 @@ int main(int argc, char *argv[])
         app.add_flag("--indent", compiler_options.indent, "Indented print ASR/AST");
         app.add_flag("--tree", compiler_options.tree, "Tree structure print ASR/AST");
         app.add_option("--pass", arg_pass, "Apply the ASR pass and show ASR (implies --show-asr)");
+        app.add_option("--skip_pass", skip_pass, "Skip an ASR pass in default pipeline");
```

Comment:
What if I want to skip multiple passes? Like loop unrolling and function inlining? I think we should treat `skip_pass` and `pass` command line arguments in the same fashion while parsing them but just ignore the pass which is present `skip_pass` vector.

Reply:
Yes, I consider it will be like the way in parse_pass_arg().
although in the issue mentioned just said ` Add an option to skip/ignore a pass in the default pipeline`
So I left it here.